src/enforce/rules/helpers/is_type/spec.js

```diff
@@ -7,93 +7,48 @@ const noop = () => undefined;
 
 describe('Tests isType helper', () => {
 
```

Comment:
The tests should now reflect `isType`'s added ability to accept multiple types as arguments (I know we test that through usage, but it could change)

Reply:
Indeed. I added tests to check the new logic.